core/src/main/java/tech/tablesaw/io/WriteOptions.java

```diff
@@ -20,17 +22,20 @@ public Destination destination() {
   public static class Builder {
 
     protected Destination dest;
+    protected boolean autoClose = true;
 
     protected Builder(Destination dest) {
       this.dest = dest;
```

Comment:
it seems like we'd want `autoClose` to be `false` in this case as well. it probably makes more sense for the default to be false and then just set `true` for `File`